### ISSUE TITLE
Build doesnt work on windows 64

### ISSUE THREAD

**Issue by cmarrero01** [nwjs/grunt-nw-builder#118]
When the build finished I try to open the ".exe" display a blank screen and when I try to open the console the application crash closing the app.

This is my config:

             nodewebkit = {
                options: {
                    platforms: ['win'],
                        buildDir: './desktop/build'
                },
                src: ['./desktop/src/**', './desktop/src/package.json'] 
            };

The cache directory have the 0.11.0-rc1.

I build manually without problems, I download the 0.10.0 for Win 32, and make this steps:

1. Make a zip of my application
2. Change the extension of app.zip to app.nw
3. Copy the app.nw to node-webkit folder
4. Open terminal and write: copy /b nw.exe+app.nw app.exe
5. Open the app.exe and Works perfectly.

But the grunt build crash.

Best
C

**Comment by robogen** [comment]
I'm having the same issue with 0.12.2. I tried the above fix of switching to 0.10.5 but it would only download the latest version. I have a Win64 bit, 16gig ram, Intel i7, Win7.

**Comment by robogen** [comment]
Managed to figure out the version control issue, and now whether through command line or manually, with version 12.3 or 10.5, I cannot get my Windows built executable running. I followed step 2b of the [nwjs wiki](https://github.com/nwjs/nw.js/wiki/how-to-package-and-distribute-your-apps) to manually build. I always get the nw:blank page. On Linux and Mac though, works just fine. Is there a dependency I'm missing?
